examples/running-tests-using-travis-and-saucelabs/README.md

```diff
@@ -0,0 +1,20 @@
+# Example: Running Tests Using Travis CI and Sauce Labs
+
+A set of sample files that help you to learn how to run TestCafe tests in the cloud using Travis CI and Sauce Labs (see [Running Tests Using Travis CI and Sauce Labs](http://devexpress.github.io/testcafe/documentation/recipes/running-tests-using-travis-ci-and-sauce-labs.html)).
+
+The following files are included:
+
+* *server.js* - runs an HTTP server for the application.
+* *index.html* - the application's main page.
+* *package.json* - the project description file. The package.json file's `dependencies` list already contains all needed packages to run the application, the `devDependencies` list contains TestCafe and Sauce Labs browser provider. The file also includes the `test` script that starts the application and runs TestCafe tests on a Sauce Labs browser.
+* *tests/index-test.js* - a simple TestCafe test.
+* *.travis.yml* - a Travis CI configuration file.
+
+## Running the example
+
+1. Create a GitHub repository and copy the sample files to it.
```

Comment:
Just write how to checkout files for further steps just give a link to recipe. No need to duplicate info here